src/pykeen/nn/node_piece.py

```diff
@@ -623,6 +623,7 @@ def __init__(
             the number of tokens for each entity.
         :param tokenizer:
             the tokenizer to use, cf. `pykeen.nn.node_piece.tokenizer_resolver`.
+            TODO: support for using both Anchor and Relation tokenizers
```

Comment:
this is a bit ambiguous - you mean together at the same time?

Reply:
yes, exactly, to use both for tokenizing based on anchors + relations

Reply:
since you added the todo in the docs, maybe use the `.. todo::` directive (https://www.sphinx-doc.org/en/master/usage/extensions/todo.html)

Reply:
ready now :)